addons/website_slides/controllers/main.py

```diff
@@ -736,6 +770,24 @@ def slide_view(self, slide, **kwargs):
         values.pop('channel', None)
         return request.render("website_slides.slide_main", values)
 
+    @http.route('/slides/slide/<int:slide_id>/share', type='http', auth="public", website=True, sitemap=False)
+    def slide_shared_view(self, slide_id, **kwargs):
+        user_slide_authorization = self._get_user_slide_authorization(slide_id)
+        status = user_slide_authorization.get('status')
```

Comment:
Same here: don't use .get (to propagate to other usage of _get_user_slide_authorization returned values)